setup-bitbucket.sh

```diff
@@ -25,9 +25,11 @@ drupal9ci_install() {
 	rsync -va --ignore-existing "$tmpdir/drupal9ci-master/dist/common/" .
 
 	# Add development dependencies to run the jobs.
+	composer self-update --2
 	COMPOSER_MEMORY_LIMIT=-1 composer require --dev	\
 		drupal/core-dev \
-		drupal/drupal-extension
+		phpspec/prophecy-phpunit:^2 \
+		drupal/drupal-extension -W
```

Comment:
LOL, I had no idea what -W was. I think that I used it before but it was the long option name.